### PR TITLE
feat: add transition for tabs

### PR DESCRIPTION
**Pull request by samobasquiat**
# Summary

Implemented a smooth transition for tab-switching for both the LeftNavigation Tabs indicator and the background of the themes tab. 

# How did you test this change?
Switching tabs in LeftNavigation and themes should have a smooth transition.

# Checklist:

- [x] I have performed a self-review of my code
- [x] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

### CODE REVIEW

**Comment by RanGojo** [comment]
Please also add same animation when switching between tabs of export modal, it seems it was missing from task description.

**Review by RanGojo — COMMENTED** [review]
Good job, please also add same animation when switching between tabs of export modal, it seems it was missing from task description.

File: widget/playground/src/components/ExportConfigModal/ExportConfigModal.styles.tsx [hunk 1]
```diff
@@ -47,17 +48,28 @@ export const StyledButton = styled(Button, {
...
 });
 
+export const BackdropTab = styled('div', {
+  width: '362px',
+  height: '$40',
+  backgroundColor: '$secondary500',
+  position: 'absolute',
+  borderRadius: '$xm',
+  inset: 0,
+  transition: 'transform 0.5s cubic-bezier(0, 0, 0.86, 1.2)',
```
Comment (nikaaru) [inline widget/playground/src/components/ExportConfigModal/ExportConfigModal.styles.tsx:70]: NIT: Considering that it is out of the container, I think a number smaller than one is better for the fourth parameter.

**Review by nikaaru — APPROVED** [review]

File: widget/playground/src/containers/StyleLayout/StyleLayout.styles.ts [hunk 2]
```diff
@@ -34,17 +34,43 @@ export const Tabs = styled('div', {
...
+      true: {
+        $$color: '$colors$background',
+        [`.${darkTheme} &`]: {
+          color: '$colors$foreground',
+        },
+        color: '$$color',
+      },
+      false: {
+        '&:hover': {
+          backgroundColor: '$info100',
```
Comment (nikaaru) [inline widget/playground/src/containers/StyleLayout/StyleLayout.styles.ts:55]: please double-check this hover in dark theme.

**Review by RanGojo — APPROVED** [review]

File: widget/playground/src/containers/StyleLayout/StyleLayout.styles.ts (hunk 2 above)
Comment (samobasquiat) [reply widget/playground/src/containers/StyleLayout/StyleLayout.styles.ts:55]: fixed.
(Note: I think in Figma, we do not have enough information about the dark playground theme in different sections.)

File: widget/playground/src/components/ExportConfigModal/ExportConfigModal.styles.tsx (hunk 1 above)
Comment (samobasquiat) [reply widget/playground/src/components/ExportConfigModal/ExportConfigModal.styles.tsx:70]: changed.

**Comment by samobasquiat** [comment]
> Good job, please also add same animation when switching between tabs of export modal, it seems it was missing from task description.

added.
@RanGojo

**Comment by RanGojo** [comment]
NIT: when I'm switching between tabs, before background color change, text color changes to white. It's not very important but I was wondering if we could fix it with some small changes in animation timings. @samobasquiat

**Review by RanGojo — APPROVED** [review]

**Comment by samobasquiat** [comment]
> NIT: when I'm switching between tabs, before background color change, text color changes to white. It's not very important but I was wondering if we could fix it with some small changes in animation timings. @samobasquiat

fixed. @RanGojo

**Review by RanGojo — APPROVED** [review]

**Pull request merged**